.github/workflows/test.yml

```
@@ -7,7 +7,7 @@ jobs:
     name: build (${{ matrix.ruby }} / ${{ matrix.os }})
     strategy:
       matrix:
-        ruby: [ '3.0', 2.7, 2.6, 2.5, 2.4, head ]
+        ruby: [ '3.1', '3.0', 2.7, 2.6, 2.5, 2.4, head ]
```

```suggestion
        ruby: [ 3.1, '3.0', 2.7, 2.6, 2.5, 2.4, head ]
```

(Reason: The quotes are about `3.0` is YAML-stringified as `"3"`. `3.1` does not have that problem.)